sources/platform/actors/publishing/monetize.mdx

```diff
@@ -71,22 +74,82 @@ You make your Actor pay-per-result and set the price to be $1/1,000 results. Dur
 Your profit is computed only from the first two users, since they are on Apify paid plans. The revenue for the first user is $50 and for the second $20, i.e., total revenue is $70. The total underlying cost is _$5 + $2 = $7_. Since your profit is 80% of the revenue minus the cost, it would be _0.8 * 70 - 7 = $49_.
 </details>
 
-#### Best practices for Pay-per-results Actors
+#### Best practices for PPR Actors
 
 To ensure profitable operation:
 
 - Set memory limits in your [`actor.json`](https://docs.apify.com/platform/actors/development/actor-definition/actor-json) file to control platform usage costs
 - Implement the `ACTOR_MAX_PAID_DATASET_ITEMS` check to prevent excess result generation
 - Test your Actor with various result volumes to determine optimal pricing
 
+### Pay-per-event pricing model
+
+The pay-per-event pricing model offers a flexible monetization option for Actors on Apify Store. Unlike the pay-per-result, PPE allows you to charge users based on specific events triggered programmatically by your Actor's code.
+
+#### PPE vs PPR
+
+Unlike PPR, which charges based on the number of results produced, PPE lets you define pricing for individual events. You can charge for specific events directly from your Actor by calling the PPE charging API. Common events include Actor start, dataset item creation, and external API calls.
+
+For a detailed comparison of pricing models, refer to our [user-facing documentation](/platform/actors/running/actors-in-store).
+
+#### How is profit computed
+
+Your profit is calculated as follows:
+
+`profit = (0.8 * revenue) - platform costs`
+
+where:
+
+- _Revenue_: The amount charged for events via the PPE [API](/api/v2/post-charge-run) or through [JS](/sdk/js/reference/class/Actor#charge)/[Python](/sdk/python/reference/class/Actor#charge) SDK. You receive 80% of this revenue.
```

Comment:
Here, I'd just link raw charging API and mention, that it's prefer to use SDKs. Because when it's like this, it creates and impressions that these 3 things are equal. But they are not, SDKs are much preferred.